### PR TITLE
`error-stack`: use `Emphasis` for all output in docs

### PR DESCRIPTION
**Pull request by indietyp**
## 🌟 What is the purpose of this PR?

The default for all output is `ColorMode::Emphasis`. To not confuse the user, this PR also changes the default output to `ColorMode::Emphasis`. It also removes the `full.png` in favor of a `pre` tag with a converted ansi to HTML output.

### CODE REVIEW

File: packages/libs/error-stack/src/fmt.rs [hunk 1]
```diff
@@ -99,7 +99,7 @@
 //!     .attach(Suggestion("try better next time!"))
 //!     .attach(Warning("unable to fetch resource"));
 //!
-//! # Report::set_color_mode(error_stack::fmt::ColorMode::Color);
+//! # Report::set_color_mode(error_stack::fmt::ColorMode::Emphasis);
```
Comment (TimDiekmann) [inline packages/libs/error-stack/src/fmt.rs:102]: Can't we entirely remove this line? 🤔

Comment (indietyp) [reply packages/libs/error-stack/src/fmt.rs:102]: True, but this makes it easier to copy/paste the whole snippet around a couple of times w/o assuming any previous color mode set 😅

File: packages/libs/error-stack/README.md [hunk 2]
```diff
@@ -95,7 +95,46 @@ fn main() -> Result<(), ExperimentError> {
 
 This will most likely result in an error and print
 
-![](https://github.com/hashintel/hash/blob/8ed55bd73045fba83a7ea2e199b31d5b829537b9/packages/libs/error-stack/assets/full.png?raw=true)
+<pre>
+Error: <b>experiment error: could not run experiment</b><span style="font-weight:normal;text-decoration:none;font-style:normal">
```
Comment (TimDiekmann) [inline packages/libs/error-stack/README.md:99]:
```suggestion
Error: <b>experiment error: could not run experiment</b>
```

File: packages/libs/error-stack/README.md [hunk 3]
```diff
@@ -95,7 +95,46 @@ fn main() -> Result<(), ExperimentError> {
...
+&#x251C;&#x2574;at <i>examples/demo.rs:51:18</i>
+&#x251C;&#x2574;unable to set up experiments
+&#x2502;
+&#x251C;&#x2500;&#x25B6; <b>invalid experiment description</b><span style="font-weight:normal;text-decoration:none;font-style:normal">
```
Comment (TimDiekmann) [inline packages/libs/error-stack/README.md:103]:
```suggestion
&#x251C;&#x2500;&#x25B6; <b>invalid experiment description</b>
```

File: packages/libs/error-stack/README.md [hunk 4]
```diff
@@ -95,7 +95,46 @@ fn main() -> Result<(), ExperimentError> {
...
+&#x2502;   &#x251C;&#x2574;at <i>examples/demo.rs:21:10</i>
+&#x2502;   &#x2570;&#x2574;experiment 2 could not be parsed
+&#x2502;
+&#x2570;&#x2500;&#x25B6; <b>invalid digit found in string</b><span style="font-weight:normal;text-decoration:none;font-style:normal">
```
Comment (TimDiekmann) [inline packages/libs/error-stack/README.md:107]:
```suggestion
&#x2570;&#x2500;&#x25B6; <b>invalid digit found in string</b>
```

File: packages/libs/error-stack/README.md [hunk 5]
```diff
@@ -95,7 +95,46 @@ fn main() -> Result<(), ExperimentError> {
...
+   5: error_stack::report::Report&lt;C&gt;::from_frame
+             at ./src/report.rs:288:25
+   6: error_stack::report::Report&lt;C&gt;::new
+             at ./src/report.rs:274:9
+   7: error_stack::context::&lt;impl core::convert::From&lt;C&gt; for error_stack::report::Report&lt;C&gt;&gt;::from
+             at ./src/context.rs:83:9
+   8: &lt;core::result::Result&lt;T,E&gt; as error_stack::result::IntoReport&gt;::into_report
+             at ./src/result.rs:203:31
+   (<b>For this example:</b> additional frames have been removed)
+</span></span></span>
```
Comment (TimDiekmann) [inline packages/libs/error-stack/README.md:134]:
```suggestion
```

**Review by TimDiekmann — COMMENTED** [review]
I love the native format for the readme, thanks!
The conversion tool, however, generated a few unneeded spans

File: packages/libs/error-stack/README.md [hunk 6]
```diff
@@ -95,7 +95,45 @@ fn main() -> Result<(), ExperimentError> {
...
+   6: error_stack::report::Report&lt;C&gt;::new
+             at ./src/report.rs:274:9
+   7: error_stack::context::&lt;impl core::convert::From&lt;C&gt; for error_stack::report::Report&lt;C&gt;&gt;::from
+             at ./src/context.rs:83:9
+   8: &lt;core::result::Result&lt;T,E&gt; as error_stack::result::IntoReport&gt;::into_report
+             at ./src/result.rs:203:31
+   (<b>For this example:</b> additional frames have been removed)
+</pre>
+
+[//]: # (![]&#40;https://github.com/hashintel/hash/blob/8ed55bd73045fba83a7ea2e199b31d5b829537b9/packages/libs/error-stack/assets/full.png?raw=true&#41;)
```
Comment (TimDiekmann) [inline packages/libs/error-stack/README.md:136]:
```suggestion
```

**Review by TimDiekmann — DISMISSED** [review]

**Review by TimDiekmann — APPROVED** [review]

**Pull request merged**